BizHawk.Client.Common/movie/tasproj/TasMovie.Editing.cs

```diff
@@ -392,7 +391,7 @@ private void ExtendMovieForEdit(int numFrames)
 			int oldLength = InputLogLength;
 			ChangeLog.AddGeneralUndo(oldLength, oldLength + numFrames - 1);
 
-			Global.MovieSession.MovieControllerAdapter.LatchSticky();
+			//Global.MovieSession.MovieControllerAdapter.LatchSticky();

```

Comment:
Maybe just delete thsi?

Reply:
I guess I can delete it permanently once everyone is happy with it going away. Honestly I couldn't grasp what it's intent was in the first place, all I know is that when appending frames to TAStudio without the movie playing that line causes errant float inputs to pop up.

Does it have a different intended use in any other scenario? I don't know.